corehq/apps/app_manager/templates/app_manager/v2/partials/case_config_shared.html

```diff
@@ -64,7 +68,18 @@
         <input type="text" class="form-control" data-bind="
             valueDefault: property.key,
             default: property.defaultKey,
-            casePropertySelect2: suggestedProperties,
+            casePropertyTypeahead: suggestedProperties,
```

Comment:
Possible bad merge? `casePropertyTypeahead` was removed in https://github.com/dimagi/commcare-hq/pull/13807